### DISCUSSION TITLE
independent vertico-vertical-mode (standard minibuffer mode) for multiform display?

### DISCUSSION THREAD

**Discussion by WuuBoLin** [minad/vertico#408] [Q&A] [answered]
Is there an independent **mode** for the standard minibuffer display for multiform, similar to `vertico-buffer-mode`, `vertico-grid-mode`, etc.?

I am currently using [vertico posframe](https://github.com/tumashu/vertico-posframe) as a vertico-multiform display. Due to posframe [being unable to be displayed in terminal (TUI)](https://github.com/tumashu/vertico-posframe/issues/16), I have set the variable `vertico-posframe-fallback-mode` to `vertico-buffer-mode` as described in the [How to let vertico-posframe work well with vertico-multiform](https://github.com/tumashu/vertico-posframe?tab=readme-ov-file#how-to-let-vertico-posframe-work-well-with-vertico-multiform) section of the README.

However, I find the buffer mode a little clunky to use. Is there any way to use the normal minibuffer mode as a fallback?

Or maybe i have to write my own function to do that so?

**Comment by minad** [comment]
I am not sure I understand the question. `vertico-buffer-mode`, `vertico-grid-mode`, `vertico-posframe-mode` all work well with `vertico-multiform-mode`.

**Reply by WuuBoLin** [reply]
The problem is that `multiform-mode` calls `vertico-posframe-mode` to display the UI in posframe. However, posframes don't work in terminals. Therefore, there is a variable called `vertico-posframe-fallback-mode` to set a mode to toggle when posframe cannot be displayed. The "normal" **vertical** mode doesn't have a separate mode to call like `vertico-grid-mode`.

**Reply by minad** [reply]
Okay, I suggest to ask about possible approaches on the vertico-posframe issue tracker.

**Comment by WuuBoLin** [comment, marked as answer]
**Problem Fixed!!** See issue https://github.com/tumashu/vertico-posframe/issues/36.
